css/styles.css

```diff
@@ -0,0 +1,281 @@
+  max-width: 93%;
+  position: relative;
+  left: 27px;
+}
+
+.top-menu > li > a {
+  text-decoration: none;
+  text-align: center;
+  color: rgb(255, 240, 238);
+  text-shadow: 0.0625em -0.0625em 0.625em rgb(52, 48, 48);
+  font-size: 0.8125em;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+.nav-form {
+  width: auto;
+  position: relative;
+  justify-content: flex-end;
+  align-items: center;
+  margin-right: -2.5em;
+  padding: 0.125em 2% 0.02em 0.02em;
+  left: 0.875em;
+  top: 0.375em;
+  overflow: hidden;
+}
+
+.nav-form > img {
+  z-index: 100;
+  max-width: 0.8125em;
+  position: absolute;
+  left: 0.5625em;
+  bottom: 1.0625em;
+}
+
+.search {
+  width: 76%;
+  height: 1.375em;
+  background: linear-gradient(to bottom, rgb(114, 100, 100), #333 50%, #8c8c8c 50%, #b3b3b3);
+  transition: all 0.5s linear;
+  border-radius: 1.375em;
+  border-top: 0.04em solid rgb(114, 100, 100);
+  border-bottom: 0.125em inset #666;
+  border-left: none;
+  overflow: hidden;
+}
+
+.search:focus {
+  background: #fff;
+  margin-left: 1.9375em;
+  width: 9.375em;
+}
+
+.titleMain {
+  max-height: 100%;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin: 3% 0 0 0;
+}
+
+.titleMain:hover {
+  cursor: pointer;
+}
+
+.titleHeader {
+  background: url("../images/h.png") center 0.25em;
+  background-repeat: no-repeat;
+  min-height: 6.25em;
+  background-size: contain;
+  width: 100%;
+}
+
+.titleHeader:invalid {
+  background: url("../images/h.png") center 0.25em;
+  background-repeat: no-repeat;
+}
+
+.titleSubHeader {
+  background: url("../images/share.png") no-repeat center;
+  background-size: contain;
+  min-height: 5em;
+  width: 100%;
+}
+
+.titleLink {
+  background: url("../images/explore.png") no-repeat center 0;
+  width: 50%;
+  height: 1.875em;
+  margin: 0.5625em;
+  padding: 0.25em;
+}
+
+.titleLink:hover {
+  border-bottom: 0.0625em solid #fff;
+  padding: 0 0.625em 0;
+}
+
+.middle-section {
+  height: 12.5em;
+  max-width: 100%;
+}
+
+.middle-section-image {
+  height: 100%;
+  width: 25%;
+  background-size: 100% 100%;
+  flex-shrink: 4;
+  background-repeat: no-repeat;
+  border: 0.03125em solid #fff;
+}
+
+.first-middle-img {
+  background-image: url("../images/iphone5s.jpg");
+}
+
+.second-middle-img {
+  background-image: url("../images/iphone5c.jpg");
+}
+
+.third-middle-img {
+  background-image: url("../images/30_years.jpg");
+}
+
+.fourth-middle-img {
+  background-image: url("../images/promo.jpg");
+}
+
+.lower-section {
+  margin-right: auto;
+  margin-left: auto;
+  max-width: 100%;
+  height: auto;
+}
+
+.bottom-box {
+  max-width: 72%;
+  min-width: 66%;
+  display: block;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.footer-txt {
+  font-size: smaller;
+  color: rgb(114, 100, 100);
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
```

Comment:
* [ ] **[OPTIONAL]**, I was checking your project on Chrome, and there the size is still too big, I changed to Firefox and saw that it is looking good there, most probable, you are using Firefox. To avoid these differences, you can better use `font_size: 1.7 rem` and that would be fine for both browsers